### PR TITLE
Update docs about formatting-types

### PR DESCRIPTION
**Pull request by imba-tjd**

### CODE REVIEW

File: docs/standard/base-types/custom-numeric-format-strings.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ You can create a custom numeric format string, which consists of one or more cus
...
 |"\\"|Escape character|Causes the next character to be interpreted as a literal rather than as a custom format specifier.<br /><br /> More information: [The "\\" Escape Character](#SpecifierEscape).|987654 ("\\###00\\#") -> #987654#|  
-|'*string*'<br /><br /> "*string*"|Literal string delimiter|Indicates that the enclosed characters should be copied to the result string unchanged.<br/><br/>More information: [Character literals](#character-literals).|68 ("# ' degrees'") -> 68  degrees<br /><br /> 68 ("#' degrees'") -> 68 degrees|  
+|'*string*'<br /><br /> "*string*"|Literal string delimiter|Indicates that the enclosed characters should be copied to the result string unchanged.<br/><br/>More information: [Character literals](#character-literals).|68 ("# ' degrees'") -> 68&nbsp;&nbsp;degrees<br /><br /> 68 ("#' degrees'") -> 68 degrees|
```
Comment (rpetrusha) [inline docs/standard/base-types/custom-numeric-format-strings.md]: The original format string is a mistake. This should be 68 ("# 'degrees'"). The output then becomes "68 degrees", and the \&nbsp; are not necessary.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thank you for making these changes, @imba-tjd. A better change for the first one, though, is to remove the additional space, as I've indicated in the comment.

File: docs/standard/base-types/custom-numeric-format-strings.md (hunk 1 above)
Comment (imba-tjd) [reply docs/standard/base-types/custom-numeric-format-strings.md]: Thanks for review. done.

**Review by rpetrusha — APPROVED** [review]
Thank you for making the additional changes, @imba-tjd. We'll merge your PR now. Your changes should be live on docs.microsoft.com in the next day or two.

**Pull request merged**